.babelrc

```diff
@@ -6,6 +6,8 @@
 		"@babel/proposal-export-namespace-from",
 		"@babel/proposal-throw-expressions",
 		"@babel/syntax-dynamic-import",
-		"babel-plugin-styled-components"
+		"babel-plugin-styled-components",
+		"@babel/plugin-proposal-nullish-coalescing-operator",
+		"@babel/plugin-proposal-optional-chaining"
```

Comment:
nice

Reply:
Already supported by typescript, so this is only necessary because we use babel-loader and fork-ts-checker-webpack-plugin rather than ts-loader